README.md

```diff
@@ -1 +1,33 @@
-# tf_cloudtrail
\ No newline at end of file
+# tf_cloudtrail
+
+Setup and manage CloudTrail
+
+### Variables
+
+#### Required
+
+* `name`: Name of CloudTrail trail.
+* `region`: AWS region. Used to find remote state.
+
+
+
+#### Optional
+
```

Comment:
Please "borrow" the same language used on the terraform docs. See: https://www.terraform.io/docs/providers/aws/r/cloudtrail.html